scripts/create_local_env_for_cil_development.sh

```diff
@@ -17,7 +17,7 @@
 set -euxo pipefail
 numpy='1.24'
 python='3.10'
-name=cil
+name=cil_test_sh
```

Comment:
probably best not to change

```suggestion
name=cil
```